### ISSUE TITLE
Add Outbound Funds to CCI Flow in CCI File

### ISSUE THREAD

**Issue by emilyhirsch-CoastalCloud** [SFDO-Community-Sprints/Snowfakery-Recipe-Templates#140]
**Recipe Name**
We want to run the OBF recipe in the target org and want it to be easier to create in org

**Org Setup**
* Command used to **create** the org: `cci org scratch` or `sfdx force:org:create` -- include any additional arguments
* Command used to **add configuration** to the org: `cci flow run dev_org` or `sfdx force:source:push` for example

The flow should be named obf_scratch to match the existing npsp_scratch. It should also install npsp.

**Comment by acrosman** [comment]
Since Snowfakery is currently embedded in CCI it seems like we should stick to assuming CCI tasks/flows not trying to also support `sfdx` at least any time soon.

Getting flows added for easy setup of some of the packages probably makes a lot of sense.

**Comment by samanthashain** [comment]
Yeah that's what this issue is really about.  I think the sfdx stuff is currently in the issue template?  I always do it the cci way!
